packages/kbn-openapi-generator/src/cli.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import yargs from 'yargs/yargs';
+import { generate } from './openapi_generator';
+import { AVAILABLE_TEMPLATES } from './template_service/template_service';
+
+export function runCli() {
+  yargs(process.argv.slice(2))
+    .command(
+      '*',
+      'Generate code artifacts from OpenAPI specifications',
+      (y) =>
+        y
+          .option('rootDir', {
+            describe: 'Root directory to search for OpenAPI specs',
+            demandOption: true,
+            string: true,
+          })
+          .option('sourceGlob', {
+            describe: 'Elasticsearch target',
+            default: './**/*.schema.yaml',
+            string: true,
+          })
+          .option('templateName', {
+            describe: 'Template to use for code generation',
+            default: 'zod_operation_schema' as const,
+            choices: AVAILABLE_TEMPLATES,
+          })
+          .showHelpOnFail(false),
+      (argv) => {
+        generate(argv).catch((err) => {
+          // eslint-disable-next-line no-console
+          console.error(err);
```

Comment:
Not 100% on usage here, but there is the `ToolingLog` package for logger usage within Kibana tooling, e.g. from the resolver generator script (which also uses console for some output, so IDK...):

https://github.com/elastic/kibana/blob/a4494815920a1aadb005aeb4711754400a1a30e4/x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts#L328-L332

Reply:
Oh, I didn't know that we have a package for logging. Thanks for bringing it up 👍 I'll look into replacing the `console.log`s with it.